goimports/runner.bash.template

```diff
@@ -3,10 +3,13 @@
 set -euo pipefail
 
 GOIMPORTS_SHORT_PATH=@@GOIMPORTS_SHORT_PATH@@
+GO_SHORT_PATH=@@GO_SHORT_PATH@@
 ARGS=@@ARGS@@
 PREFIX_DIR_PATH=@@PREFIX_DIR_PATH@@
 PREFIX_BASE_NAME=@@PREFIX_BASE_NAME@@
 
+GO_BIN_DIR=$(dirname $(readlink ${GO_SHORT_PATH}))
```

Comment:
How about this new approach with `readlink` and without `$PWD`? `readlink` was also used for the `goimports` location.
If that is not good enough, I can check tomorrow for a better solution, but this is really basic bash. If that would not work, I am not sure how such bash runner rules could work.

Reply:
If you want a solution with this path helper, do you have an example somewhere?

Reply:
If this works then I guess I'm happy with the approach. Thanks a lot for fixing the issue!

Reply:
> If this works then I guess I'm happy with the approach. Thanks a lot for fixing the issue!

Thanks. Works great.